src/App.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { GoodList } from './components/GoodList/GoodList';
```

Comment:
Ensure that the `GoodList` component exists in the specified path and is correctly implemented to handle the `goods` prop.